client/src/Components/Footer/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import CopyrightIcon from '@material-ui/icons/Copyright';
+import useStyles from './style';
+
+function Footer() {
+  const classes = useStyles();
+  return (
+    <Container maxWidth="lg" className={classes.footer}>
+      <div>
+        <CopyrightIcon />
+        <p>2021 all copyright reserved</p>
+      </div>
+    </Container>
```

Comment:
* we need the footer to get all window width
* why you use div for wrapper the container here is a wrapper element 
*  replace the container with a footer element, for [accessibility](https://medium.com/alistapart/writing-html-with-accessibility-in-mind-a62026493412) issues 
``` 
 <footer className={classes.footer}>
      <CopyrightIcon className={classes.icon} />
      <p>2021 all copyright reserved</p>
    </footer> ```